build/root/WORKSPACE

```diff
@@ -29,6 +29,14 @@ http_archive(
     urls = mirror("https://github.com/coreos/etcd/releases/download/v%s/etcd-v%s-linux-amd64.tar.gz" % (ETCD_VERSION, ETCD_VERSION)),
 )
 
+http_archive(
```

Comment:
and then for extra extra credit you could move these into `build/workspace.bzl` and follow a pattern similar to the other dependencies there (like CNI and CRI) to loop over all the relevant architectures.

Reply:
Done.